docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -42,6 +42,21 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
 
 ### Parameters
 
+* **grouping_expression**
+
+    Specifies the criteria based on which the rows are grouped together. The grouping of rows is performed based on
+    result values of the grouping expressions. A grouping expression may be a column alias, a column position
+    or an expression.
```

Comment:
not relate to this PR, but we can give some examples to make it easy to understand.
```
A grouping expression may be a column alias like `GROUP BY a`, a column position
like `GROUP BY 0`, or an expression like `GROUP BY a + b`.
```

Reply:
Done